okapi-core/src/main/resources/log4j2.properties

```diff
@@ -1,19 +1,25 @@
 status = error
 name = PropertiesConfig
-packages = org.folio.okapi.common.logging
 
 filters = threshold
 
 filter.threshold.type = ThresholdFilter
 filter.threshold.level = info
 
-appenders = console
+appender.full.type = Console
+appender.full.name = FULL
+appender.full.layout.type = PatternLayout
+appender.full.layout.pattern = %d{HH:mm:ss} [${map:requestid}] [${map:tenantid}] [${map:userid}] [${map:moduleid}] %-5p %-20.20C{1} %m%n
 
-appender.console.type = Console
-appender.console.name = STDOUT
-appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = %d{HH:mm:ss} [$${FolioLoggingContext:requestid}] [$${FolioLoggingContext:tenantid}] [$${FolioLoggingContext:userid}] [$${FolioLoggingContext:moduleid}] %-5p %-20.20C{1} %m%n
+appender.omitted.type = Console
+appender.omitted.name = OMITTED
```

Comment:
For me `omitted` is somewhat misleading because the log entry is not omitted. What about `basic`?

Reply:
Maybe `empty` since it's the pattern for when there's no values inside the brackets. It took me a while to grok "omitted" as well.

Reply:
OK. I also slightly prefer "empty" over "basic":